core/templates/dev/head/pages/exploration_editor/editor_tab/StateContentEditorDirective.js

```diff
@@ -28,21 +28,22 @@ oppia.directive('stateContentEditor', [
         };
       },
       scope: {
-        getOnSaveContentFn: '&onSaveContentFn'
+        getOnSaveContentFn: '&onSaveContentFn',
+        onSaveStateContent: '=',
+        onSaveContentIdsToAudioTranslations: '=',
+        isInitialState: '&'
       },
       templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
         '/pages/exploration_editor/editor_tab/' +
         'state_content_editor_directive.html'),
       controller: [
         '$scope', '$uibModal', 'stateContentService', 'EditabilityService',
-        'EditorFirstTimeEventsService', 'ExplorationInitStateNameService',
-        'EditorStateService', 'stateContentIdsToAudioTranslationsService',
-        'COMPONENT_NAME_CONTENT',
+        'EditorFirstTimeEventsService', 'COMPONENT_NAME_CONTENT',
+        'stateContentIdsToAudioTranslationsService',
```

Comment:
You mean stateContentIdsToAudioTranslationsService? But then, all services in StatePropertyServices should also be capitalized, right? I think this could be fine.

Reply:
Yes, all services in general should be capitalized.

Reply:
Done.